netmesh/model/netmesh/netmesh.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package netmesh;
+
+enum ApiVersion {
+    NETMESH_API_VERSION_1 = 0;
+};
+
+enum PayloadType {
```

Comment:
Do we want to dictate payloads types as an enumerable here? There are other payloads that we may see, e.g. Geneve, GRE, LISP. There may be others we don't know about. We could extend this list over time, but it may be better to leave this as a fillable label with the schema for that label managed outside of the proto.

Reply:
I'll change the payload value to a string for now.